### ISSUE TITLE
v2.2.0 fast, but...

### ISSUE THREAD

**Issue by robtesch** [knuckleswtf/scribe#141]
- [x] **I confirm that I have read and attempted the tips in the [Troubleshooting Guide](https://scribe.readthedocs.io/en/latest/troubleshooting.html).**

**What happened?**
1. I updated to the latest version of scribe (2.2.0)
2. Then I ran `php artisan scribe:generate --force` 
3. But I saw...

two of my endpoints that require unique values seem to have hit validation errors (XYZ has already been taken).
for what it's worth, both of these endpoints were "update" endpoints, and they previously worked just fine before the parallel requests thing you added.

instead of...

**Screenshots and stack traces:**

**My environment:**
 - PHP version (from `php -v`): 7.4.11
 - Framework (Laravel/Lumen): Laravel
 - Laravel/Lumen version (from `composer show laravel/framework` or `composer show lumen/framework`): v8.14.0
 - Scribe version (from `composer show knuckleswtf/scribe`): 2.2.0

**My Scribe config (minus the comments):** Would rather not post here but if necessary I will do so.

**Additional info:** As I say, I love this package and it's being really good, things are definitely also faster (although for me they weren't slow before) but I think something with the DB transactions this time round has messed with a few of the endpoints.

**Comment by shalvah** [comment]
> two of my endpoints that require unique values seem to have hit
validation errors (XYZ has already been taken).

Gonna need more details.

**Comment by shalvah** [comment]
Also, tell me about your db connections? Do you have multiple connections that use the same db?

**Comment by robtesch** [comment]
Sorry, I will get more details asap.

Effectively these are stock standard Laravel "unique" validation rules with
"ignore" appended for the specific ID of the model being updated.

My DB connections are fully standard single mysql connection (out of the
box settings).

**Comment by shalvah** [comment]
> Sorry, I will get more details asap. Effectively these are stock standard Laravel "unique" validation rules with "ignore" appended for the specific ID of the model being updated. My DB connections are fully standard single mysql connection (out of the box settings).

Why not just copy-paste a sample from your code and what you get rather than trying to describe it in English?😕 Right now I'm still not sure of your situation.

There's a reason there's a field for screenshots and stack traces.

**Comment by ncatanchin** [comment]
I'm also getting this problem following upgrade to `v2.2` - 

```
Illuminate\Database\QueryException: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'bhackett@example.com' for key 'users_email_unique'
```

It appears that the doc generation process is persisting data to the database. The problem is showing up in my tests, which also use faker.

**Comment by shalvah** [comment]
Thanks. I'm investigating this. In the meantime, I recommend using a testing database to be on the safe side. You can run with a `--env` option to set an env file.

**Comment by ncatanchin** [comment]
@shalvah you likely have seen this one, but just in case - https://github.com/knuckleswtf/scribe/issues/147
seems the issue reporter was able to go down the debug track a fair bit

**Comment by shalvah** [comment]
This should be fixed in 2.4.0, but might be a breaking change if you use more than one transaction. See https://github.com/knuckleswtf/scribe/blob/master/CHANGELOG.md#240-monday-30-november-2020
